modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -442,84 +418,89 @@ See below:
-and [tokens](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs). The
-components now support the `cs` prop, but otherwise the API has not changed.
-
-> #### Visual Breaking Change
->
-> The `border` color for `SecondaryButton` on the `focus` state is now `transparent` and may cause a
-> small visual difference in visual tests.
-
-### Card
-
-**PR:** [#2471](https://github.com/Workday/canvas-kit/pull/2471)
-
-`Card` and its subcomponents have been refactored to use our new
-[styling utilities](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-modifiers#createstyles-api)
-and [tokens](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs). `Card`
-no longer extends the `Box` component, however, it still supports Emotion `styled` and style props.
-
-### Checkbox
-
-**PR:** [#2542](https://github.com/Workday/canvas-kit/pull/2542)
+## Component Updates
 
-`Checkbox` now uses
+### Styling API and CSS Tokens
+
+**PRs:** [#2666](https://github.com/Workday/canvas-kit/pull/2666),
+[#2471](https://github.com/Workday/canvas-kit/pull/2471),
+[#2542](https://github.com/Workday/canvas-kit/pull/2542),
+[#2570](https://github.com/Workday/canvas-kit/pull/2570),
+[#2442](https://github.com/Workday/canvas-kit/pull/2442),
+[#2472](https://github.com/Workday/canvas-kit/pull/2472),
+[#2685](https://github.com/Workday/canvas-kit/pull/2685),
+[#2615](https://github.com/Workday/canvas-kit/pull/2615),
+[#2699](https://github.com/Workday/canvas-kit/pull/2699),
+[#2546](https://github.com/Workday/canvas-kit/pull/2546),
+[#2570](https://github.com/Workday/canvas-kit/pull/2570),
+[#2620](https://github.com/Workday/canvas-kit/pull/2620),
+[#2583](https://github.com/Workday/canvas-kit/pull/2583),
+[#2567](https://github.com/Workday/canvas-kit/pull/2567),
+[#2455](https://github.com/Workday/canvas-kit/pull/2455)
+
+Several components have been refactored to use new
 [Canvas Tokens](https://workday.github.io/canvas-tokens/?path=/docs/docs-getting-started--docs) and
-our
-[new styling utilities](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-modifiers#createstyles-api).
-The component now supports the `cs` prop, but otherwise the API has not changed. It should behave
-identically as it did in previous versions.
+[styling API](https://workday.github.io/canvas-kit/?path=/docs/styling-basics--create-modifiers#createstyles-api).
+The React interface **has not changed**, but CSS variables are now used for dynamic properties.
 
-### Combobox
+The following components are affected: `PrimaryButton`, `SecondaryButton`,
+`TertiaryButton`,`DeleteButton`, `Card`, `CheckBox`, `Combobox`, `CountBadge`,
+`FormField (Preview)`, `Svg`, `SystemIcon`, `SystemIconCircle`, `AccentIcon`, `AppletIcon`,
+`Graphic`, `LoadingDots`, `Radio (Preview)`, `Select`, `StatusIndicator`, `Switch`, `Table` and
+`Text`
```

Comment:
Is there a better way to have these listed out? Not sure this is the best format as it is hard to scan.